core/tests/ts-integration/tests/api/snapshots-creator.test.ts

```diff
@@ -0,0 +1,84 @@
+import { TestMaster } from '../../src/index';
+import fs from 'fs';
+import * as zlib from 'zlib';
+import { snapshots_creator } from 'zk/build/run/run';
+describe('Snapshots API tests', () => {
```

Comment:
Nit: A blank line between `import`s and main code would improve readability.

Reply:
fixed